uPortal-webapp/src/main/resources/properties/jgroups.xml

```diff
@@ -42,13 +42,14 @@
          thread_pool.keep_alive_time="30000"
     />
 
-    <!-- user database for discovery -->
+    <!-- User database for discovery;  initialize_sql to blank prevents
+         table creation, which we handle with Hibernate via dbloader -->
     <JDBC_PING
             connection_url="${hibernate.connection.url}"
             connection_username="${hibernate.connection.username}"
             connection_password="${hibernate.connection.password}"
             connection_driver="${hibernate.connection.driver_class}"
-            initialize_sql="${jgroupsInitSql}"
+            initialize_sql=""
```

Comment:
Looks good. `initialize_sql` needs to be set to empty string to skip default action of creating a table with MS SQL Server syntax